test/functional/driver/driver-e2e-specs.js

```diff
@@ -83,30 +83,6 @@ describe('XCUITestDriver', function () {
         simsAfter.should.equal(simsBefore);
       });
 
-      it('with udid: uses sim and shuts it down afterwards', async () => {
```

Comment:
@imurchie what do you think about just updating this to be a test of the new cap?